src/Apps/Sell/Routes/ThankYouRoute.tsx

```diff
@@ -43,22 +43,26 @@ export const ThankYouRoute: React.FC<ThankYouRouteProps> = props => {
           <Join separator={<Spacer y={4} />}>
             <Join separator={<Spacer y={2} />}>
               <SubmissionStepTitle>
-                {isSubmitted
-                  ? "Thank you for submitting your artwork"
+                {!!isSubmitted
+                  ? "Thank you for submitting additional information"
```

Comment:
I think we need to replace "Thank you for listing your artwork" with "Thank you for submitting additional information". This is the text that shows up after the tier 2 flow (when the submission has the state "APPROVED").

Reply:
You're right

Reply:
It was all the other way around. Fixed now. Updated the screenshots as well